### ISSUE TITLE
intermittent loss of scan results

### ISSUE THREAD

**Issue by MarvinT** [CameronRedmore/memory-deck#52]
Is there any way for memory deck to reload previous scanned results if it re-opens still attached to same process? Accidentally pressing B with memory deck open makes you lose all your results. Also, all the results seem to be lost at other times that I haven't quite figured out the trigger for.

**Comment by CerebrawlDeck** [comment]
Yeah it seems to lose the scan after just a few minutes, going back to showing the default 0, extremely frustrating.
